types/busboy/v0/busboy-tests.ts

```diff
@@ -0,0 +1,261 @@
+import Busboy = require('busboy');
+
+// test type exports
+type Constructor = Busboy.BusboyConstructor;
+type Config = Busboy.BusboyConfig;
+type Headers = Busboy.BusboyHeaders;
+type BB = Busboy.Busboy;
+type Events = Busboy.BusboyEvents;
+
+new Busboy({}); // $ExpectError
+const busboy = Busboy({ headers: { 'content-type': 'foo' } }); // $ExpectType Busboy
+new Busboy({ headers: { 'content-type': 'foo' } }); // $ExpectType Busboy
```

Comment:
is this even possible under v1? It looks only factory method is exported, returning instance or throwing an error.

Reply:
forget this one, I've missed versioning